app/views/handlers/confirmation_email.py

```diff
@@ -21,7 +25,7 @@ class ConfirmationEmailLimitReached(Exception):
     pass
 
 
-class ConfirmationEmailFulfilmentRequestPublicationFailed(Exception):
+class ConfirmationEmailTaskCreationFailed(Exception):
```

Comment:
The class name should be changed back since we decided for it to be a fulfilment request.